### DISCUSSION TITLE
Error in running the contracts folder for slither analysis

### DISCUSSION THREAD

**Discussion by IshwaryaLakshmanan** [crytic/slither#1934] [Q&A]
Issue facing is RecursionError: maximum recursion depth exceeded while calling a Python object why this is occurring tried to set the depth level to 10000, but it doesn't work when running the below command. 
` slither . --solc-remaps @openzeppelin=node_modules`

What is the resolution for this error?

**Comment by 0xalpharush** [comment]
This sounds similar to open bug reports and will be resolved by https://github.com/crytic/slither/pull/1832 in the upcoming release.
